### PR TITLE
[Polygon ID] Update PID links

### PR DESCRIPTION
**Pull request by otto-mora**
Update Polygon ID links for tutorials page and discord.

### CODE REVIEW

File: docs/polygonid/overview.md [hunk 1]
```diff
@@ -17,9 +17,17 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Based on the principles of Self-Sovereign Identity (SSI) and cryptography, Polygon ID is a decentralised and permissionless identity framework for web2 and web3 applications. The SSI framework lets individuals own and control their identities. 
 
+:::info
+
+For the latest technical documentation and tutorials, please head over to our [<ins>Polygon ID tutorials page</ins>](/docs/zkEVM/overview).
```
Comment (ashucoder9) [inline docs/polygonid/overview.md:22]: Why are we using zkEVM link here?

File: docs/polygonid/overview.md [hunk 2]
```diff
@@ -17,9 +17,17 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+:::
+
 :::tip Contact
 
-You can always refer to our [<ins>twitter page</ins>](https://twitter.com/0xpolygonid) to stay updated with the Polygon ID latest development.  
+You can ask questions in our [<ins>discord for Polygon ID</ins>](https://discord.com/channels/635865020172861441/1052090291743637504)
+
+Also please refer to our [<ins>twitter page</ins>](https://twitter.com/0xpolygonid) to stay updated with the latest developments on Polygon ID.
```
Comment (ashucoder9) [inline docs/polygonid/overview.md:30]:
```suggestion
Also please refer to our [<ins>Twitter page</ins>](https://twitter.com/0xpolygonid) to stay updated with the latest developments on Polygon ID. 
```

Comment (ashucoder9) [reply docs/polygonid/overview.md:30]: We can club these two admonitions together.

**Review by ashucoder9 — CHANGES_REQUESTED** [review]
A few queries and changes.

File: docs/polygonid/overview.md (hunk 1 above)
Comment (otto-mora) [reply docs/polygonid/overview.md]: Good catch. Thanks I corrected to our Github Tutorials for Polygon ID.

**Comment by otto-mora** [comment]
> A few queries and changes.

Thanks Ash! Corrected one and accepted your suggestion for the other. Suggest we go ahead and merge now.

**Review by DannyS03 — APPROVED** [review]

**Pull request merged**